.gitignore

```diff
@@ -7,4 +7,4 @@ tests/logs
 .idea
 tests/test-output
 .env
-src/test/resources/local.properties
+tests/src/test/resources/local.properties
```

Comment:
how come this was changed? was the file location changed?

Reply:
Location is not changed. File wasn't ignored in the earlier commit. Checking with path with content root instead of from source.

Reply:
ok so it was wrong earlier and this is the right path

Reply:
That's correct.